go.mod

```diff
@@ -0,0 +1,69 @@
+module github.com/signalfx/signalfx-go-tracing
```

Comment:
Is this intentional?  Upstream isn't using mod and I don't believe we want to declare any target dependencies.

Reply:
May be worthwhile to add these to .gitignore

Reply:
Not intentional. Had to run tests locally and forgot to remove.

Reply:
Fixed.